src/ticket/controllers/ticket.controller.ts

```
@@ -1,25 +1,95 @@
+  ApiOperation,
+} from '@nestjs/swagger';
+import { UniversalJwtAuthGuard } from 'src/auth/guards/universal.jwt-auth.guard';
+import { UniversalJwtRequest } from 'src/auth/types/universal.jwt-request';
 import { CreateTicketDto } from 'src/ticket/dtos/create-ticket.dto';
-import { TicketService } from 'src/ticket/services/ticket.service';
+import { TicketNotExistException } from 'src/ticket/exceptions/ticket-not-exist.exception';
 
 @Controller('tickets')
 export class TicketController {
-  constructor(private ticketService: TicketService) {}
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)